Formula/re2.rb

```diff
@@ -1,9 +1,9 @@
 class Re2 < Formula
   desc "Alternative to backtracking PCRE-style regular expression engines"
   homepage "https://github.com/google/re2"
-  url "https://github.com/google/re2/archive/2021-02-02.tar.gz"
-  version "20210202"
-  sha256 "1396ab50c06c1a8885fb68bf49a5ecfd989163015fd96699a180d6414937f33f"
+  url "https://github.com/google/re2/archive/refs/tags/2021-04-01.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/google/re2/archive/2021-04-01.tar.gz"
```
Should be the same file, but would be more consistent with regards to style in other current formulae.

Reply:
Sure; I just used this one because it's what you get when you click on "Source Code (.tar.gz)" on their releases page.